enrolments/tests/utils/utils.py

```diff
@@ -5,12 +5,23 @@
 fake = Faker()
 Faker.seed(0)
 
+ENROLMENT_STATUSES = [
+    "Waiting to enrol",
+    "Registered",
+    "Confirmed",
+    "Completed",
+    "No show",
+    "Drop out",
+    "Waitlisted",
+]
```

Comment:
can we reference the statuses on the enrolment model directly? similarly to `Session.SEASON_CHOICES` below!

Reply:
ya I'll make it a list comprehension